File: help/pt_pt/exercicios_da_ficha_4/Ficha04_009/Ficha04_009.cs
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

using System.IO;
using System;
public class Ficha04_009 {

    //Main Function
    static void Main() {
        Console.Write("Verificar se a soma de dois valores reais" );
        Console.Write("\né igual a um terceiro\n" );
        Console.Write("Real 1 \t: " );
        double r1 = double.Parse(Console.ReadLine());
        Console.Write("Real 2 \t: " );
        double r2 = double.Parse(Console.ReadLine());
        Console.Write("Real 3 \t: " );
        double r3 = double.Parse(Console.ReadLine());
        Console.Write(r1 + " + " + r2 + " == " + r3 + " => " );
        if(r1 + r2 == r3 ){
            Console.Write("Verdadeiro" );
        }else{
            Console.Write("Falso" );
        }
    }

}
//C#  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Mon, 2022 Sep 26 15:32

File: help/pt_pt/exercicios_da_ficha_4/Ficha04_009/Ficha04_009.java
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

import java.util.Scanner;

public class Ficha04_009 {

    static Scanner keyb = new Scanner(System.in);//keyboard

    //Main Function
    public static void main( String[] args) {
        System.out.print("Verificar se a soma de dois valores reais" );
        System.out.print("\né igual a um terceiro\n" );
        System.out.print("Real 1 \t: " );
        double r1 = Double.valueOf(keyb.nextLine());
        System.out.print("Real 2 \t: " );
        double r2 = Double.valueOf(keyb.nextLine());
        System.out.print("Real 3 \t: " );
        double r3 = Double.valueOf(keyb.nextLine());
        System.out.print(r1 + " + " + r2 + " == " + r3 + " => " );
        if(r1 + r2 == r3 ){
            System.out.print("Verdadeiro" );
        }else{
            System.out.print("Falso" );
        }
    }

}
//Java  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Mon, 2022 Sep 26 15:32
